OctoberCms.gitignore

```diff
@@ -0,0 +1,19 @@
+/bootstrap/compiled.php
+/vendor
+composer.phar
+.DS_Store
```

Comment:
We use [`Global\macOS.gitignore`](https://github.com/github/gitignore/blob/master/Global/macOS.gitignore) to track macOS-specific rules. Please omit this rule from your template to prevent unnecessary duplication.